### PR TITLE
SLU-02 | Missing Return Statement

### PR DESCRIPTION
**Pull request by asalzmann**
## What is the purpose of the change
missing errors at audit commit hash
msg_server_add_validator.go#L16-18: 16
msg_server_submit_tx.go#L222-224: 222
msg_server_submit_tx.go#L242-244: 242

## Brief Changelog

- add errors

## Testing and Verifying

CI + integration

## Documentation and Release Note

  - Does this pull request introduce a new feature or user-facing behavior changes? no
  - Is a relevant changelog entry added to the `Unreleased` section in `CHANGELOG.md`? no
  - How is the feature or change documented? audit

### CODE REVIEW

File: x/stakeibc/keeper/msg_server_submit_tx.go [hunk 1]
```diff
@@ -281,6 +281,7 @@ func (k Keeper) GetLightClientTimeSafely(ctx sdk.Context, connectionID string) (
 	conn, found := k.IBCKeeper.ConnectionKeeper.GetConnection(ctx, connectionID)
 	if !found {
 		k.Logger(ctx).Error(fmt.Sprintf("invalid connection id, \"%s\" not found", connectionID))
+		return 0, false
```
Comment (riley-stride) [inline x/stakeibc/keeper/msg_server_submit_tx.go:284]: Glad we patched this one, the LC was updating on line 287 even when connection id was invalid

**Review by riley-stride — APPROVED** [review]
LGTM
glad we caught these

File: x/stakeibc/keeper/msg_server_submit_tx.go (hunk 1 above)
Comment (asalzmann) [reply x/stakeibc/keeper/msg_server_submit_tx.go:284]: 😬

Comment (riley-stride) [reply x/stakeibc/keeper/msg_server_submit_tx.go:285]: Unfort I don't think it was the cause of the 50+ LC update txs per block issue, that seems to be more a function of either (a) having too many LCs or (b) updating all LCs on a fixed cadence (e.g. epochly)
I don't have a good answer for it yet [](edited-with-Axolo)

Comment (asalzmann) [reply x/stakeibc/keeper/msg_server_submit_tx.go:285]: Might be a mix of both? Not sure it's an issue [](edited-with-Axolo)

**Pull request merged**